### ISSUE TITLE
[Bug Report] "input" event of el-input-number is emitted twice

### ISSUE THREAD

**Issue by hairui219** [ElemeFE/element#11544]
### Element UI version
2.4.1

### OS/Browsers version
Windows 10   Chrome 67 / Electron 2.0.2

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/4jbq5hvx/](https://jsfiddle.net/4jbq5hvx/)

### Steps to reproduce
See console output

### What is Expected?
The event should only be emitted once.

### What is actually happening?
The event is now emitted twice.

###  Workaround

A workaround would be listen to the "change" event instead of "input"

**Comment by atilkan** [comment]
There is no **input** event on **el-input-number** element. Try with **change** event.
**Updated fiddle. ->** https://jsfiddle.net/4jbq5hvx/1/
**Docs. ->** http://element.eleme.io/#/en-US/component/input-number#inputnumber

**Comment by hairui219** [comment]
The ```input``` event is the default event for ```v-model``` directive.

Here is the reference
[```v-model``` on a component uses ```value``` as the prop and ```input``` as the event](https://vuejs.org/v2/guide/components-custom-events.html#Customizing-Component-v-model)

However, I can confirm that it didn't cause problems. The ```change``` event worked well and  the ```watch``` event of v-model prop in parent component didn't trigger twice.

**Comment by atilkan** [comment]
I guess they did not care input event and provided change event. What are you trying to accomplish?

**Comment by hairui219** [comment]
In fact it is a hint for me that if an event is triggered more than desired, it must be some programming problem in the component (based on my own experience). 

It is more of a quality concern now since there is no problem using the component.

FYI: I have used separate event (eg. ```change``` or ```update```) for component updates in the past. But I found the ```input``` event could also get the job done without adding the separate event. So I changed my way to listen for the ```input``` event by the DRY (Don't repeat yourself) principle.
